### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by Mark90**
* Adapt docker-compose setup for usage in GH Actions workflow
* Integration tests for create/modify/terminate
* Schedule to run tests every other week and report to Slack

### CODE REVIEW

**Review by mrijk — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -43,28 +43,6 @@ cp -av examples/*add_user_and_usergroup* migrations/versions/schema
...
 ```
 
-Now point your browser to `http://localhost:3000/` and have a look around. 
-You can use the `New Process` button to create subscription on the defined 
+Now point your browser to `http://localhost:3000/` and have a look around.
+You can use the `New Process` button to create subscription on the defined
 products.
-
-
-## Integration tests
```
Comment (tjeerddie) [inline README.md:51]: we do want some description of the tests

Comment (Mark90) [reply README.md:51]: I moved it into tests/integration_tests/README.md

Comment (Mark90) [reply README.md:51]: Because it's not very relevant to put in the main readme (which is focused on people exploring the orchestrator)

**Pull request merged**